vcsh.in

```diff
@@ -529,6 +530,17 @@ which() {
 	fi
 }
 
+locate() {
+	if [ ! -f "$VCSH_COMMAND_PARAMETER" ]; then
+		fatal "'$VCSH_COMMAND_PARAMETER' does not exist" 1
+	fi
+	for VCSH_REPO_NAME in $(list); do
+		if get_files "$VCSH_COMMAND_PARAMETER" | @GREP@ -q .; then
```

Comment:
This `grep` feels *very* superfluous. Did you look into what the return being given are and if they might suffice for this logic?